src/toil/cwl/cwltoil.py

```diff
@@ -4261,8 +4262,15 @@ def main(args: Optional[list[str]] = None, stdout: TextIO = sys.stdout) -> int:
     runtime_context.move_outputs = "leave"
     runtime_context.rm_tmpdir = False
     runtime_context.streaming_allowed = not options.disable_streaming
+    if options.cachedir is not None:
+        runtime_context.cachedir = os.path.abspath(options.cachedir)
+        # Automatically bypass the file store to be compatible with cwltool caching
```

Comment:
```suggestion
        # Automatically bypass the file store to be compatible with cwltool caching
        # Otherwise, the CWL caching code makes links to temporary local copies
        # of filestore files and caches those.
```